testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/sessionlimits/KcOidcUserSessionLimitsBrokerTest.java

```diff
@@ -0,0 +1,11 @@
+package org.keycloak.testsuite.sessionlimits;
+
+import org.keycloak.testsuite.broker.BrokerConfiguration;
+import org.keycloak.testsuite.broker.KcOidcBrokerConfiguration;
+
```

Comment:
Since your test uses `testingClient.server().run`, I think it is needed to also include annotation `@AuthServerContainerExclude(REMOTE)` at the beginning of the test class. The same applies also for `KcSamlUserSessionLimitsBrokerTest` . See some other classes (EG. `UserSessionLimitsTest`) for the inspiration. Without this annotation, the test will likely fail in the pipeline when executed with `auth-server-remote` .

NOTE: It is possible to add this annotation only to `AbstractUserSessionLimitsBrokerTest` and it might exclude it from both subclasses. But I am not 100% sure if having this on abstract class is ok to skip the tests on subclasses. IMO safer to add this to subclasses. But you can try to investigate if you want and if you don't like separate annotation for every subclass.